src/main/java/com/minecolonies/achievements/AbstractSizeAchievement.java

```diff
@@ -0,0 +1,112 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.block.Block;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
+ * Abstract
+ * 
+ * @author Isfirs
+ * @since 0.2
+ */
+public abstract class AbstractSizeAchievement extends AbstractAchievement

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2160)